apps/api/docker-compose.yml

```
@@ -0,0 +1,31 @@
+# Use root/example as user/password credentials
+version: "3.1"
+
+services:
+    mongo:
+        image: mongo
+        restart: always
+        ports:
+            - "27017:27017"
+        environment:
+            MONGO_INITDB_ROOT_USERNAME: root
+            MONGO_INITDB_ROOT_PASSWORD: example
+
+    mongo-express:
+        image: mongo-express
+        restart: always
+        ports:
+            - "8081:8081"
+        environment:
+            ME_CONFIG_MONGODB_ADMINUSERNAME: root
+            ME_CONFIG_MONGODB_ADMINPASSWORD: example
+
+            ME_CONFIG_MONGODB_URL: mongodb://root:example@mongo:27017/
+
+    fastapi-dev:
+        network_mode: host
+        build: .
+        ports:
+            - "8000:8000"
+        environment:
+            MONGODB_URI: mongodb://root:example@127.0.0.1:27017
```

Chore: we can reference the container directly as you did within `mongo-express` instead of using 127.0.0.1
```suggestion
            MONGODB_URI: mongodb://root:example@mongo:27017
```